src/types/offer/interfaces/offer.interfaces.ts

```diff
@@ -13,3 +13,20 @@ export interface OfferResponse {
   bio: string
   price: number
 }
+
+export interface OfferResponce {
+  _id: string
+  price: string
+  proficiencyLevel: string[]
+  description: string
+  languages: string[]
+  authorRole: string
+  authorFirstName: string
+  authorLastName: string
+  authorAvgRating: number
+  authorId: string
+  subjectId: string
+  categoryId: string
+  createdAt: string
+  updatedAt: string
```

Comment:
i guess we should think about to move fields like _id, createdAt, updatedAt to interface and extends from it, because we use them in almost all models

Reply:
Fixed